src/test/java/com/google/api/generator/spring/SpringAutoConfigClassComposerTest.java

```diff
@@ -189,8 +190,9 @@ public void generatePropertiesTest() {
           + "  @Bean\n"
           + "  @ConditionalOnMissingBean\n"
           + "  public EchoClient echoClient(\n"
-          + "      CredentialsProvider credentialsProvider,\n"
-          + "      TransportChannelProvider defaultTransportChannelProvider)\n"
+          + "      @Qualifier(\"googleCredentials\") CredentialsProvider credentialsProvider,\n"
```

Comment:
Totally missed this: We want to add this `@Qualifier` so that when multiple services are present, there won't be conflict in which `CredentialsProvider` bean it takes. So need to rename `googleCredentials()` to `[serviceName]Credentials()` and match the name here. (same as `defaultEchoTransportChannelProvider()`)

Reply:
Thanks for catching this - will address in a separate PR!